modules/avatar/react/stories/stories_avatarButton.tsx

```diff
@@ -0,0 +1,60 @@
+/// <reference path="../../../../typings.d.ts" />
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import withReadme from 'storybook-readme/with-readme';
+
+import {AvatarButton} from '../index';
+import README from '../README.md';
+
+const IMAGE_URL = 'https://s3-us-west-2.amazonaws.com/design-assets-internal/avatars/lmcneil.png';
+
+storiesOf('AvatarButton', module)
```

Comment:
Can you put all of these under Avatar (`storiesOf('Avatar/AvatarButton', module)`)? You'll have to update the other story file as well.
```
Avatar/
  Default/
    Light
    Dark
    Image
  AvatarButton/
    Light
    Dark
    Image
```

Reply:
cool